docs/sources/alerting/manage-notifications/manage-contact-points/configure-notifiers.md

```diff
@@ -0,0 +1,68 @@
+---
+description: Notifier configuration guide
+draft: true
+keywords:
+  - Grafana
+  - alerting
+  - guide
+  - notifications
+  - notifiers
+  - contact points
+title: Configure notifiers
+weight: 100
+---
+
+# Configure notifiers
+
+Configuring notifiers in Grafana allows you to receive notifications for your alert rules in your preferred communication channels. Each notifier has its own configuration options and setup process, but in general, the process involves providing an API key or webhook URL.
+
+Once configured, you can use notifiers as part of your contact points to receive notifications whenever your alert changes its state. In this section, we'll cover the basic steps to configure your notifiers, so you can start receiving real-time alerts and stay on top of your monitoring data.
```

Comment:
Is a notifier different to a contact point? I thought contact points were notifiers? Can you configure your notifiers to receive alert rules in different states - or only when firing?

Reply:
In which case, I'd write "Once you've configured your contact points, you'll receive  notifications whenever your alert rule changes state.

Reply:
> Is a notifier different to a contact point? I thought contact points were notifiers?

I agree it's confusing! A contact point is basically a set of notifiers used together. For example, we have [docs](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/webhook-notifier/) for configuring the Webhook notifier, which we can use in a contact point alongside other notifiers.

In upstream Alertmanager terms:
- Contact point -> Receiver.
- Notifier -> Integration.

I just checked and realized that we use the word "integrations" A LOT to refer to notifiers in our docs and in Grafana. It would be great if the naming was more consistent, so I think I'll change this to use that term.

Reply:
> Can you configure your notifiers to receive alert rules in different states - or only when firing?

You can receive resolved alerts too, this is an opt-out.
You can also receive _NoData_ and _Error_ notifications, which are not the same as _Firing_.